source/FreeRTOS_ARP.c

```diff
@@ -1338,6 +1340,24 @@ static BaseType_t prvFindCacheEntry( const MACAddress_t * pxMACAddress,
             }
         }
     }
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Create and send an ARP request packet.
+ *
+ * @param[in] ulIPAddress A 32-bit representation of the IP-address whose
+ *                         physical (MAC) address is required.
+ */
+    void FreeRTOS_OutputARPRequest( uint32_t ulIPAddress )
+    {
+        const NetworkEndPoint_t * pxEndPoint = FreeRTOS_FindEndPointOnNetMask( ulIPAddress, 12 );
```

Comment:
Please add a comment mentioning the below assumptions :
1. IPv4 endpoints belonging to same physical interface can have same subnet
2. IPv4 endpoints belonging to different physical interface in the system will have a different subnet.